go.sum

```diff
@@ -67,14 +69,18 @@ github.com/Cerebellum-Network/chainbridge-utils v1.0.7-0.20240123112715-6db11339
 github.com/Cerebellum-Network/chainbridge-utils v1.0.7-0.20240123112715-6db113397ae7/go.mod h1:hNZH7DOtyHQqz22U/B1ecP5qzElQVD1DVM7NNB6Q38s=
 github.com/Cerebellum-Network/chainbridge-utils v1.0.7-0.20240205092257-877ee052801b h1:qoGaL+iBejT5hK3sJD2DMtOHsRxlpLy7YssGRCi6AL0=
 github.com/Cerebellum-Network/chainbridge-utils v1.0.7-0.20240205092257-877ee052801b/go.mod h1:F6zsU7xTRF8CHOyXAZWvMGoLJ5XV8MRc5R0t3qw3oaY=
+github.com/Cerebellum-Network/chainbridge-utils v1.2.0-rc1 h1:K1B6znmrH/eiprqJdJewKdTN6alCx6tERlzsicSDR1E=
+github.com/Cerebellum-Network/chainbridge-utils v1.2.0-rc1/go.mod h1:g8f0YUDoFbl1j2W7OHbncAwUwKJ+xO2FYd0q6MPb0mM=
 github.com/Cerebellum-Network/go-substrate-rpc-client/v7 v7.0.1-rc h1:HGg29xHzn64oYqlsPDGfi2eHJHYWp9hqzVjjW1k58/0=
 github.com/Cerebellum-Network/go-substrate-rpc-client/v7 v7.0.1-rc/go.mod h1:jWRJ6bf690StnOgY9hxZ8bR8CPDk3+dd6aGU0StFKM8=
-github.com/Cerebellum-Network/go-substrate-rpc-client/v8 v8.0.1-rc h1:U/o8tNQzsjRz5lQS1Zdfhm4ot/vfviAD3mVaEI35egE=
-github.com/Cerebellum-Network/go-substrate-rpc-client/v8 v8.0.1-rc/go.mod h1:3NiCQzqkio9HRDnP3Kck3RWFrF+pa7x/UYhD5vjmttA=
-github.com/Cerebellum-Network/go-substrate-rpc-client/v8 v8.0.2-rc h1:R1ZegZvpBaCaZ8nYih8qZRiFqzoJYeNG0JVLnlEtleA=
-github.com/Cerebellum-Network/go-substrate-rpc-client/v8 v8.0.2-rc/go.mod h1:3NiCQzqkio9HRDnP3Kck3RWFrF+pa7x/UYhD5vjmttA=
-github.com/Cerebellum-Network/go-substrate-rpc-client/v8 v8.0.4-rc h1:Rd0Px+IrX4M2woDxl49QCsT9l6CY1At4pFNKlhWTqpA=
-github.com/Cerebellum-Network/go-substrate-rpc-client/v8 v8.0.4-rc/go.mod h1:3NiCQzqkio9HRDnP3Kck3RWFrF+pa7x/UYhD5vjmttA=
+github.com/Cerebellum-Network/go-substrate-rpc-client/v9 v8.0.1-rc h1:U/o8tNQzsjRz5lQS1Zdfhm4ot/vfviAD3mVaEI35egE=
+github.com/Cerebellum-Network/go-substrate-rpc-client/v9 v8.0.1-rc/go.mod h1:3NiCQzqkio9HRDnP3Kck3RWFrF+pa7x/UYhD5vjmttA=
+github.com/Cerebellum-Network/go-substrate-rpc-client/v9 v8.0.2-rc h1:R1ZegZvpBaCaZ8nYih8qZRiFqzoJYeNG0JVLnlEtleA=
+github.com/Cerebellum-Network/go-substrate-rpc-client/v9 v8.0.2-rc/go.mod h1:3NiCQzqkio9HRDnP3Kck3RWFrF+pa7x/UYhD5vjmttA=
+github.com/Cerebellum-Network/go-substrate-rpc-client/v9 v8.0.4-rc h1:Rd0Px+IrX4M2woDxl49QCsT9l6CY1At4pFNKlhWTqpA=
+github.com/Cerebellum-Network/go-substrate-rpc-client/v9 v8.0.4-rc/go.mod h1:3NiCQzqkio9HRDnP3Kck3RWFrF+pa7x/UYhD5vjmttA=
```

Comment:
v9 refers to v8 tag. Is it correct?

Reply:
No, v9 should refer to tag [v9.0.2](https://github.com/Cerebellum-Network/go-substrate-rpc-client/releases/tag/v9.0.2)

Reply:
This is Go's package manager weirds, I'll try to clean up